src/main/webapp/services/assignUserService.js

```diff
@@ -34,6 +34,7 @@ export default class AssignUserService extends BaseService {
     };
 
     this.updateDialog('assignIssue', template, userData);
+    setTimeout(() => this.closeDialog('assignIssue'), 3000);
```

Comment:
Is there any indication for the user that will happen? I don't think the user is expecting this behavior.

Reply:
The last dialog after assigning the issue is a dead end, as the user can't do anything besides closing the dialog. This is a suggestion from Vince, so we can ask the UX team what should be the better behavior.